### PR TITLE
Private/pedro/hyperlink improvements add buttons

### PR DESCRIPTION
**Pull request by pedropintosilva**
It would be cool to have a hyperlink popup that is not so big in size but has bigger font size plus a couple of actions so the user can edit, remove hyperlink without exiting the popup and trigger the context menu

This PR does that (without sending the uno command yet):
image
^ we will then need to add new less complex icon for remove hyperlink icon to be used here

I need help here, my initial feeling was to do something like this (and obviously it is wrong):
```javascript
editBtn.onclick = this._map.sendUnoCommand('.uno:EditHyperlink');
```

### CODE REVIEW

File: browser/src/layer/tile/CanvasTileLayer.js [hunk 1]
```diff
@@ -2417,8 +2417,21 @@ L.CanvasTileLayer = L.Layer.extend({
 		if (!url.startsWith('#')) {
 			var link = L.DomUtil.createWithId('a', 'hyperlink-pop-up');
 			link.innerText = url;
+			var editBtn = L.DomUtil.createWithId('div', 'hyperlink-pop-up-edit');
+			editBtn.setAttribute('style', 'display: inline-block; margin-left: 12px; vertical-align: bottom; height: 18px;');
+			var imgEditBtn = L.DomUtil.create('img', 'hyperlink-pop-up-editimg', editBtn);
+			imgEditBtn.setAttribute('src', L.LOUtil.getImageURL('baseline-edit.svg'));
+			imgEditBtn.setAttribute('width', 18);
+			imgEditBtn.setAttribute('height', 18);
+			var removeBtn = L.DomUtil.createWithId('div', 'hyperlink-pop-up-remove');
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/layer/tile/CanvasTileLayer.js:2432]: fixed in the last commit, when element is passed by html text - events are not copied

[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/src/layer/tile/CanvasTileLayer.js:2442]: thanks!

**Comment by pedropintosilva** [comment]
https://user-images.githubusercontent.com/65948705/201329291-578cfb0c-d414-40f5-80ab-01c4c08389d2.mp4

[x] Buttons work (copy, edit, remove)
[x] Add tooltips
[x] Add hover states
[] `leaflet-popup-content` 's width is set accounting for the buttons otherwise a long URL forces to hide one or more buttons. Buttons should be always visible. If there is no space to have both (url and buttons) url should get the ellipsis instead

**Review by gokaysatir — APPROVED** [review]
Thanks for the improvements @pedropintosilva :)

**Pull request merged**